util/db/user/user.go

```diff
@@ -32,3 +32,37 @@ func UserById(ctx context.Context, id string, db *pgxpool.Pool) (map[string]stri
 		"fullName": fullName,
 	}, nil
 }
+
+// CheckEmail checks if an email exists in users table
+func CheckEmail(ctx context.Context, email string, db *pgxpool.Pool) (bool, error) {
+	query := `SELECT EXISTS(SELECT 1 FROM users WHERE email = $1)`
+
+	var exists bool
+	err := db.QueryRow(ctx, query, email).Scan(&exists)
+	if err != nil {
+		return false, err
+	}
+
+	return exists, nil
+}
+
+// GetUserIDByEmail returns user ID for given email
+func GetUserIDByEmail(ctx context.Context, email string, db *pgxpool.Pool) (string, error) {
```

Comment:
Rename to `UserIDFromEmail`

https://google.github.io/styleguide/go/best-practices.html#naming-conventions